Userland/Libraries/LibJS/Runtime/Temporal/PlainYearMonthPrototype.cpp

```diff
@@ -208,6 +209,23 @@ JS_DEFINE_NATIVE_FUNCTION(PlainYearMonthPrototype::to_string)
     return js_string(vm, *string);
 }
 
+// 9.3.18 Temporal.PlainYearMonth.prototype.toLocaleString ( [ locales [ , options ] ] ), https://tc39.es/proposal-temporal/#sec-temporal.plainyearmonth.prototype.tolocalestring
+JS_DEFINE_NATIVE_FUNCTION(PlainYearMonthPrototype::to_locale_string)
```

Comment:
how about adding a `// NOTE: This is the minimum toLocaleString implementation for engines without ECMA-402.`?

Reply:
Great catch, thanks!